apps/tlon-mobile/src/ErrorBoundary.tsx

```diff
@@ -0,0 +1,42 @@
+import crashlytics from '@react-native-firebase/crashlytics';
+import React, { Component, ErrorInfo, ReactNode } from 'react';
+import { Text, View } from 'react-native';
+
+interface ErrorBoundaryProps {
+  children: ReactNode;
+}
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+  error: Error | null;
+}
+
+class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  constructor(props: ErrorBoundaryProps) {
+    super(props);
+    this.state = { hasError: false, error: null };
+  }
+
+  static getDerivedStateFromError(error: Error): ErrorBoundaryState {
+    return { hasError: true, error };
+  }
+
+  componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+    crashlytics().recordError(error);
```

Comment:
maybe only attempt this if not in dev environment?

Reply:
Crashlytics apparently doesn't send anything unless we're running in prod.